### PR TITLE
milestone-three

### PR DESCRIPTION
**Pull request by RoyNyaga**
In this branch, we added a wide and color props to the button component then styled the calculator.

### CODE REVIEW

File: src/styles/App.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#app-wrapper-div {
```
Comment (Oghenebrume50) [inline src/styles/App.css]: This should use flex box see this
Also the display and buttonpanel components, please make changes accordingly

File: src/components/Button.js [hunk 2]
```diff
@@ -4,13 +4,20 @@ import PropTypes from 'prop-types'
 import '../styles/Button.css'
 
 const Button = (props) => {
+  // var allowRegister = age > 18
+  console.log(props.wide)
```
Comment (Oghenebrume50) [inline src/components/Button.js]: Please remove commented codes and delete console.log,  these things are only for debugging they should not make it to production.

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -1,38 +1,40 @@
...
+        <Button name='X' wide={false} />
       </div>
       <div id='group-three' className='row'>
-        <Button name='4' classNaming='generalBtn' />
-        <Button name='5' classNaming='generalBtn' />
-        <Button name='6' classNaming='generalBtn' />
-        <Button name='-' classNaming='generalBtn' />
+        <Button name='4' color='##ffffff' wide={false} />
+        <Button name='5' color='##ffffff' wide={false} />
+        <Button name='6' color='##ffffff' wide={false} />
```
Comment (Oghenebrume50) [inline src/components/ButtonPanel.js]: When a property is boolean, you do not need to explicitly pass it false, not passing it at all would do the trick, so here the wide prop can be removed and for all other occurrences were it is set to false

File: src/styles/Button.css [hunk 4]
```diff
@@ -1,9 +1,7 @@
-.generalBtn {
-	width: 100px;
-	height: 60px;
+.normal {
```
Comment (Oghenebrume50) [inline src/styles/Button.css]: The button component should have borders see this

File: src/styles/App.css (hunk 1 above)
Comment (RoyNyaga) [reply src/styles/App.css]: Ok will do that

**Comment by RoyNyaga** [comment]
Used display flex property on app buttonPanel and button component. Removed commented codes and console logs. Removed wide prop on buttons which shouldn't display a size twice as normal so as to make it resolve to false. Added borders to the button element.

**Comment by RoyNyaga** [comment]
yuuupppppiiiiiiii will do just that before merging.

**Pull request merged**